AmplifyPlugins/API/AWSAPICategoryPluginTestCommon/Model/Todo.swift

```diff
@@ -8,14 +8,11 @@
 import Foundation
 import Amplify
 
-/// TODO: Replace with AmplifyTestCommon's DataStore's geneated model
-/// This model corresponds to the resources created for the Todo graphQL endpoint. As a developer, this is
-/// hand-written from the codegen that created the API.swift that depends on AppSync sdk.
 struct Todo: Decodable {
    let typename: String
    let id: String
    let name: String
-   let description: String
+   let description: String?
```

Comment:
this is because the default schema changed, and the decision made here is to go with the new default schema so the test code needs to be updated